### PR TITLE
Use JavaScript classes

### PR DESCRIPTION
**Pull request by omar-muhamad**
- [ ] Add book class with constructor.
- [ ] Include add and remove methods in the class.
- [ ] Make these methods to add and remove books data to DOM and local storage.
- [ ] Add some basic styling and check linters errors.

### CODE REVIEW

**Comment by omar-labana** [comment]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, which have no professional README presentation of your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+
+![screenshot](Website-screenshoot.png)
+
+## Built With
+
+- HTML.
+- CSS.
+- JavaScript.
+- Linters.
+
```
Comment (Dipeshtwis) [inline README.md:22]: - [x] I would recommend mentioning the steps on the README on how to start the project. It's very important to include the steps if the user/recruiter wants to run the project locally but doesn't get proper steps. It gives a bad impression to the recruiter about you. Please update that.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Omar-Muhamad and @audukehinde, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Uses Javascript classes. :heavy_check_mark: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
Hi @Omar-Muhamad and @audukehinde, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**